sections/semantics-embedded-content.include

```diff
@@ -3267,30 +3270,43 @@ My &lt;img src="heart.png" alt="heart"&gt; breaks.
     </div>
 
     <div class="example">
-  Sometimes the entire point of the image is that a textual
-  description is not available, and the user is to provide the
-  description. For example, software that displays images and
-  asks for alternative text precisely for the purpose of then
-  writing a page with correct alternative text. Such a page could
-  have a table of images, like this:
+      Sometimes the entire point of the image is that a textual
+      description is not available, and the user is to provide the
+      description. For example, software that displays images and
+      asks for alternative text precisely for the purpose of then
+      writing a page with correct alternative text. Such a page could
+      have a table of images, like this:
 
-  <pre highlight="html">
-&lt;table&gt;
-  &lt;tr&gt;&lt;tr&gt; &lt;th&gt; Image &lt;th&gt; Description&lt;tr&gt;
-  &lt;td&gt;
-  &lt;figure&gt;
-  &lt;img src="2421.png"&gt;
-  &lt;figcaption&gt;Image 640 by 100, filename 'banner.gif'&lt;/figcaption&gt;
-  &lt;/figure&gt;
-  &lt;td&gt; &lt;input name="alt2421"&gt;
-  &lt;tr&gt;
-  &lt;td&gt; &lt;figure&gt;
-  &lt;img src="2422.png"&gt;
-  &lt;figcaption&gt;Image 200 by 480, filename 'ad3.gif'&lt;/figcaption&gt;
-  &lt;/figure&gt;
-  &lt;td&gt; &lt;input name="alt2422"&gt;
-  &lt;/table&gt;
-    </pre>
+      <xmp highlight="html">
+        <table>
+          <tr>
+            <th>Image</th>
+            <th>Description</th>
+          </tr>
+          <tr>
+            <td>
+              <figure>
+                <img src="2421.png">
+                <figcaption>Image 640 by 100, filename 'banner.gif'</figcaption>
+              </figure>
+            </td>
+            <td>
+              <input name="alt2421">
```

Comment:
should we add closing tag to input element?

Reply:
I think so.